mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -260,11 +260,49 @@ void CSaveLocSystem::LevelInitPreEntity()
     }
 }
 
+bool CSaveLocSystem::LoadStartMarks()
+{
+    if (m_pSavedLocsKV)
+    {
+        DevLog("Loaded startmarks from %s!\n", SAVELOC_FILE_NAME);
+
+        if (!m_pSavedLocsKV->IsEmpty())
+        {
+            KeyValues *kvMapSavelocs = m_pSavedLocsKV->FindKey(gpGlobals->mapname.ToCStr());
+            if (kvMapSavelocs && !kvMapSavelocs->IsEmpty())
+            {
+                KeyValues *kvStartMarks = kvMapSavelocs->FindKey("startmarks");
+                if (!kvStartMarks)
+                    return false;
+
+                CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
+
+                if (!pPlayer)
+                    return false;
+
+                FOR_EACH_SUBKEY(kvStartMarks, kvStartMark)
+                {
+                    int track = atoi(kvStartMark->GetName());
+
+                    auto c = new SavedLocation_t;
```

Comment:
Try to use descriptive variable names so that other code readers (potentially future you!) aren't solving an algebraic expression to understand your code 😄 

```suggestion
                    auto startMark = new SavedLocation_t;
```

Reply:
This was a symptom of copying a few lines from above, but `auto c` has now been replaced by `SavedLocation_t *startMark`